src/components/CardLists/planner.js

```diff
@@ -27,6 +30,10 @@ function CardlistPlanner() {
 									delete
 								</StyledDeleteButton>
 							</StyledCardHeader>
+							<StyledCardBody>
+								<StyledCardImage src={card.image}></StyledCardImage>
```

Comment:
To use a` <image />` element is not wrong but I would suggest you to use the `<Image />  component of next js`, which brings some advantages. 
Here's a link to the documentation: [next/image](https://nextjs.org/docs/api-reference/next/image)

Reply:
thanks a lot for your feedback, I can learn so much from it! 🥳